### PR TITLE
Paket template comment

### PR DESCRIPTION
**Pull request by MorganPersson**
Comments in paket.template files by using `#`.

### CODE REVIEW

**Comment by forki** [comment]
Could you please also add // as comments? IIRC we support both in dependencies file

File: src/Paket.Core/TemplateFile.fs [hunk 1]
```diff
@@ -224,11 +231,12 @@ module internal TemplateFile =
     let private fromReg = Regex("from (?<from>.*)", RegexOptions.Compiled)
     let private toReg = Regex("to (?<to>.*)", RegexOptions.Compiled)
     let private isExclude = Regex("\s*!\S", RegexOptions.Compiled)
-
+    let private isComment = Regex("\s*#", RegexOptions.Compiled)
```
Comment (forki) [inline src/Paket.Core/TemplateFile.fs]: you already have the same/similar regex above

**Comment by MorganPersson** [comment]
support for // added

**Comment by forki** [comment]
very cool. thx

**Pull request merged**